int32 LevelI
---
int32 LevelF
